### PR TITLE
Add Retry

### PR DESCRIPTION
**Pull request by waigani**

### CODE REVIEW

File: diffparser.go [hunk 1]
```diff
@@ -36,6 +38,24 @@ type DiffRange struct {
...
+	for delay := time.Millisecond; ; delay *= 2 {
+		attempt++
+		if err := f(); err == nil {
+			return attempt, nil
+		}
+		if delay >= 10*time.Second {
+			delay = 10 * time.Second
+		}
+		select {
+		case <-time.After(delay):
```
Comment (waigani) [inline diffparser.go:52]: This is going to create a new time.After object on each iteration and leak mem for the duration of the for loop.

Comment (waigani) [reply diffparser.go:52]: @codelingobot capture Ticker in for select

Make one ticker as opposed to making a new timer each iteration.

Comment (waigani) [reply diffparser.go:52]: @codelingo capture test rule

Comment (waigani) [reply diffparser.go:52]: @codelingo capture Ticker in for select

Make one ticker as opposed to making a new timer each on iteration.

Comment (Daanikus) [reply diffparser.go:52]: @codelingobot capture Ticker in for select

Make one ticker as opposed to making a new timer each on iteration.

Comment (Daanikus) [reply diffparser.go:52]: @CodeLingoBot capture Ticker in for select

Make one ticker as opposed to making a new timer each on iteration.

**Pull request closed without merging**